src/components/app/ListOfPublishedProfiles.css

```diff
@@ -0,0 +1,82 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+.publishedProfiles {
+  /* Box */
+  width: 95%;
+  box-sizing: border-box;
+  padding: 3em 1em;
+  border: 1px solid #ccc;
+  margin: auto;
+
+  /* Other */
+  background: #fff;
+  border-radius: 3px;
+  box-shadow: 0 5px 25px #0b1f50;
+  font-size: 13px;
+  line-height: 1.5;
+}
+
+.publishedProfilesList {
+  padding: 0;
+  margin: 0;
+  list-style: none;
+}
+
+.publishedProfilesDate {
+  position: absolute;
+}
+
+.publishedProfilesName {
+  overflow: hidden;
+  text-overflow: ellipsis;
+  white-space: nowrap;
+}
+
+.publishedProfilesName,
+.publishedProfilesList .profileMetainfoSummary {
+  padding-left: 10em;
+}
+
+.publishedProfilesListItem {
+  padding: 5px 0;
+}
+
+.publishedProfilesLink {
+  display: block;
+  padding: 2px 5px;
+  color: inherit;
+  text-decoration: none;
+}
+
+.publishedProfilesLink:hover {
+  background: var(--grey-90-a10);
```

Comment:
This appears to be a bit darker than the spec. I'd prefer to have it lighter. I don't think this particularly blocks accessibility to have it lighter, as it's more of a style effect. Tabbing with the keyboard still has the very clear photon outline styling.

Reply:
Thanks, I'll double check.

Reply:
I used the -a05 flavor instead, this is indeed better.
Plus the buttons I'll add should use -a10 according to the spec, and they appear on hover only, so I'll need to find a way to make them stand out of the hover style... maybe with a shadow.